cardano-wasm/example/cardano-api.d.ts

```diff
@@ -0,0 +1,99 @@
+// cardano-api.d.ts
+
+export default initialize;
+
+/**
+ * Initializes the Cardano API.
+ * @returns A promise that resolves to the main `CardanoAPI` object.
+ */
+declare function initialize(): Promise<CardanoAPI>;
```

Comment:
suggestion: use british english, instead of american english?
```suggestion
declare function initialise(): Promise<CardanoAPI>;
```